cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -310,18 +310,16 @@ func runAgent(ctx context.Context,
 		}
 	}
 
-	isOnlyDefaultConfigPath := len(fOtelConfigs) == 1 && fOtelConfigs[0] == paths.YamlConfigPath
-
 	if len(c.Inputs) != 0 && len(c.Outputs) != 0 {
 		log.Println("creating new logs agent")
 		logAgent := logs.NewLogAgent(c)
 		// Always run logAgent as goroutine regardless of whether starting OTEL or Telegraf.
 		go logAgent.Run(ctx)
 
-		// If only the default CWAgent config yaml is provided and does not exist, then ASSUME
+		// If only a single YAML is provided and does not exist, then ASSUME the agent is
 		// just monitoring logs since this is the default when no OTEL config flag is provided.
 		// So just start Telegraf.
-		if isOnlyDefaultConfigPath {
+		if len(fOtelConfigs) == 1 {
 			_, err = os.Stat(fOtelConfigs[0])
 			if errors.Is(err, os.ErrNotExist) {
```

Comment:
can we add a warning message here? or it won't be necessary? I'm just thinking about a case where there is a typo in a given path.